### PR TITLE
IN-7 feat: add select

### PR DESCRIPTION
**Pull request by web-serge**

### CODE REVIEW

File: src/components/select/select.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import type { Meta } from "@storybook/react"
+
+import { Select } from "./select"
+
+const meta = {
+  component: Select,
+  parameters: {
```
Comment (incubatorLead) [inline src/components/select/select.stories.tsx:7]: Please, remove ` layout: "centered"`  configuration from here and move it to general configuration of storybook

File: src/components/select/select.module.scss [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+.viewport {
+  display: flex;
+  flex-direction: column;
+}
+
+.item {
+  cursor: pointer;
+  padding: 6rem 12rem;
```
Comment (incubatorLead) [inline src/components/select/select.module.scss:76]: Please do not use rem for box model properties. Need to use PX for such purpose. In general we should use rem only for font-size property

File: src/components/select/select.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  className?: string
+  onValueChange?: (items: string) => void
+  options: Options[]
+  placeholder?: string
+  value?: string
+} & React.ComponentPropsWithoutRef<typeof SelectPrimitive.Root>
+
+export const Select = React.forwardRef<React.ElementRef<typeof SelectPrimitive.Root>, Props>(
+  (
+    { children, className, disabled, onValueChange, options, placeholder, value, ...props },
```
Comment (incubatorLead) [inline src/components/select/select.tsx:21]: I suggest to use `...restProps` insted of `...props `
image

File: src/components/select/select.stories.tsx (hunk 1 above)
Comment (web-serge) [reply src/components/select/select.stories.tsx]: fixed

File: src/components/select/select.module.scss (hunk 2 above)
Comment (web-serge) [reply src/components/select/select.module.scss]: fixed

File: src/components/select/select.tsx (hunk 3 above)
Comment (web-serge) [reply src/components/select/select.tsx]: fixed

**Review by incubatorLead — APPROVED** [review]

File: src/components/select/select.tsx [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+    return (
+      <SelectPrimitive.Root
+        disabled={disabled}
+        name={name}
+        onValueChange={onValueChange}
+        {...restProps}
+      >
+        <Typography
+          as={"label"}
+          className={clsx(s.label)}
```
Comment (SergFediaev) [inline src/components/select/select.tsx:66]: `className={s.label}`

File: src/components/select/select.tsx [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+  id?: string
+  labelText?: ReactElement
+  minWidth?: number
+  name: string
+  onValueChange?: (items: string) => void
+  options: Options[]
+  placeholder?: string
+} & React.ComponentPropsWithoutRef<typeof SelectPrimitive.Root>
+
+export const Select = React.forwardRef<React.ElementRef<typeof SelectPrimitive.Root>, Props>(
```
Comment (SergFediaev) [inline src/components/select/select.tsx:27]: Same here:
`export const Select = forwardRef<ElementRef<typeof SelectPrimitive.Root>, Props>(`

File: src/components/select/select.tsx [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+type Props = {
+  className?: string
+  id?: string
+  labelText?: ReactElement
+  minWidth?: number
+  name: string
+  onValueChange?: (items: string) => void
+  options: Options[]
+  placeholder?: string
+} & React.ComponentPropsWithoutRef<typeof SelectPrimitive.Root>
```
Comment (SergFediaev) [inline src/components/select/select.tsx:25]: React. can be omitted:
`} & ComponentPropsWithoutRef<typeof SelectPrimitive.Root>`

File: src/components/select/select.tsx [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+      >
+        <Typography
+          as={"label"}
+          className={clsx(s.label)}
+          htmlFor={selectId}
+          variant={"regular_text_14"}
+        >
+          {labelText}
+          <SelectPrimitive.Trigger
+            className={clsx(s.trigger, className && className)}
```
Comment (Abrosimov96) [inline src/components/select/select.tsx:72]: className={clsx(s.trigger, className)}
If className is undefined it is ignored

File: src/components/select/select.tsx (hunk 4 above)
Comment (web-serge) [reply src/components/select/select.tsx:66]: fix

File: src/components/select/select.tsx (hunk 5 above)
Comment (web-serge) [reply src/components/select/select.tsx:27]: fix

File: src/components/select/select.tsx (hunk 7 above)
Comment (web-serge) [reply src/components/select/select.tsx:72]: fix

**Pull request merged**